src/components/security_manager/test/security_manager_test.cc

```diff
@@ -101,11 +101,12 @@ class SecurityManagerTest : public ::testing::Test {
   /*
    * Wrapper for fast emulate recieve SecurityManager::OnMessageReceived
    */
-  void call_OnMessageReceived(const uint8_t* const data, uint32_t dataSize,
+  void call_OnMessageReceived(const uint8_t* const data,
+                              uint32_t dataSize,
                               const ServiceType serviceType) {
     const ::protocol_handler::RawMessagePtr rawMessagePtr(
-        new ::protocol_handler::RawMessage(key, protocolVersion, data, dataSize,
-                                           serviceType));
+        new ::protocol_handler::RawMessage(
```

Comment:
@AMikhniev , double check all places in this PR where you use `new`. Replace with `MakeShared` where possible. In this particular place the call of `new` can be avoided.